app/src/main/java/de/tum/in/tumcampusapp/models/barrierfree/BarrierfreeContact.java

```diff
@@ -0,0 +1,86 @@
+package de.tum.in.tumcampusapp.models.barrierfree;
+
+import de.tum.in.tumcampusapp.adapters.SimpleStickyListHeadersAdapter;
+
+/**
+ * The model used to display contact infromation in barrier free page
+ */
+public class BarrierfreeContact implements SimpleStickyListHeadersAdapter.SimpleStickyListItem{
+//    private int id;
+    private String name;
+    private String phone;
+    private String email;
+    private String faculty;
+    private String tumonlineID = "";
+
+    public BarrierfreeContact(String name, String phone, String email, String faculty){
+        this.name = name;
+        this.phone = phone;
+        this.email = email;
+        this.faculty = faculty;
+    }
+
+    public BarrierfreeContact(){
+
+    }
+
+//    public int getId() {
+//        return id;
+//    }
+
+    public String getName(){
+        return name;
+    }
+
+    public String getPhone(){
+        return phone;
+    }
+
+    public String getEmail(){
+        return email;
+    }
+
+    public String getFaculty() {
+        return faculty;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setPhone(String phone) {
+        this.phone = phone;
+    }
+
+    public void setEmail(String email) {
```

Comment:
No, they are not necessary. I create them because the constructor is full of string arguments and I am confused. So I create setters to avoid ordering mistakes.